apiserver/logtransfer.go

```diff
@@ -29,7 +29,9 @@ func newMigrationLoggingStrategy(ctxt httpContext, fileLogger io.Writer) Logging
 // Authenticate checks that the user is a controller superuser and
 // that the requested model is migrating. Part of LoggingStrategy.
 func (s *migrationLoggingStrategy) Authenticate(req *http.Request) error {
-	st, err := s.ctxt.stateForMigration(req)
+	// Require MigrationModeNone because logtransfer happens after the
+	// model proper is completely imported.
+	st, err := s.ctxt.stateForMigration(req, state.MigrationModeNone)
```

Comment:
Unfortunately not at present. Once the migration hits SUCCESS, the MigrationTarget.Active API is called on the target which sets the migration mode to MigrationModeNone, allowing workers to start and the model to function.

I guess we could add a "post import" phase but that will require some updates to various bits of the code which currently only trigger on MigrationModeNone. It is a bit unfortunate that the current situation means the /migrate/logtransfer import is available during normal operation but I'm not too worried about it at this stage.

Xtian: can you please add a card to the board in the model migration section which covers this? We can tidy it up later.